packages/oedax_v1/contracts/impl/libauction/AuctionStatus.sol

```diff
@@ -35,6 +35,7 @@ library AuctionStatus
         returns (uint)
     {
         assert(s.askAmount > 0);
+        // Q: Should this be s.S.mul(s.bidAmount).mul(s.bidBaseUnit) / s.askAmount / s.askBaseUnit;?
```

Comment:
the bidAmount is the satoshi value, so to convert it to humber readable unit, 

bid = bidAmount/bidBaseUnit
ask = askAmount/askBaseUnit

price = S*bidAmount/bidBaseUnit/(askAmount/askBaseUnit)
= S*bidAmount*askBaseUnit/bidBaseUnit/askAmount

so the original one should be correct